ext/ffi/lib.rs

```diff
@@ -1483,6 +1647,20 @@ unsafe fn do_ffi_callback(
           v8::Number::new(scope, result as f64).into()
         }
       }
+      NativeType::Struct(_) => {
+        let size = native_type.get_size().unwrap();
```

Comment:
You can actually get this struct size by passing the `cif` into the `do_ffi_callback` function and doing something like this:
```rs
let cif_types = std::slice::from_raw_parts(cif.arg_types, cif.nargs as usize);
let n_arg = cif_types[n];
let n_arg = n_arg.as_ref().unwrap();
let size = n_arg.size;
```

This way you won't need to have the size calculation function at all.